how-to-handle-history-mode-files.md

```diff
@@ -0,0 +1,220 @@
+#### What is History Mode
+
+History mode allows us to capture every version of each record processed by the fivetran connectors.
+In order to keep all versions of the record, we have introduced three new system columns for tables with history mode enabled.
+
+
+Column | Type | Description
+--- | --- | ---
+_fivetran_active | Boolean | TRUE if it is the currently active record. FALSE if it is a historical version of the record. Only one version of the record can be TRUE.
+_fivatran_start | TimeStamp | The time when the record was first created or modified in the source.
+_fivetran_end | TimeStamp | The value for this column depends on whether the record is active. If the record is not active(`_fivetran_active`=FALSE), then `_fivetran_end` value will be `_fivetran_start` of the next version of the record minus 1 millisecond. If the record is deleted, then the value will be the same as the timestamp of delete operation. If the record is active(`_fivetran_active`=TRUE), then `_fivetran_end` is the max allowed value that we can set for a TIMESTAMP column.
+
+
+#### Points to remember in history mode
+
+- In WriterBatchRequest we pass a new optional field HistoryMode which indicates connector is in history mode or not. In this HistoryMode field, we pass `deleted_column` column name which we need to modify only if it is present in destination.   
```

Comment:
```suggestion
- In `WriterBatchRequest`, we pass a new optional field HistoryMode which indicates connector is in history mode or not. In this HistoryMode field, we pass `deleted_column` column name which we need to modify only if it is present in destination.   
```
Shouldn't it be `history_mode` or something similar? Why the discrepancy in the letter case for field names?

Reply:
This is a new field Type we pass in proto interface. It shouldn't follow camelcase. 
Field Type should follow this case `HistoryMode` and Field Name should follow camelcase i.e. `history_mode`.
Here we are providing details go Field Type